sale_coupon_domain_product_discount/README.rst

```diff
@@ -14,14 +14,14 @@ Coupon Promotion Product Domain Discount
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--promotion-lightgray.png?logo=github
-    :target: https://github.com/OCA/sale-promotion/tree/13.0/sale_coupon_domain_product_discount
+    :target: https://github.com/OCA/sale-promotion/tree/15.0/sale_coupon_domain_product_discount
```

Comment:
This change is not correct. Please run again the README generator to go back to 13.0 definitions.

Reply:
corrected, thanks